content/en/docs/tutorials/add-prebuilt-driver-config.md

```diff
@@ -0,0 +1,79 @@
+
+## Generate the configuration
+
+You can generate and contribute configurations for your OS, as follows.
+
+Let's start by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [test-infra](https://github.com/falcosecurity/test-infra) repository.
+From the `driverkit` directory of the repository, run:
+
+```shell
+make generate -e TARGET_DISTRO=<TARGET_DISTRO> -e TARGET_KERNEL=<TARGET_KERNEL>
+```
+
+filling it with:
+* `TARGET_DISTRO`: the Linux distribution of the target host.
+* `TARGET_KERNEL`: the Linux kernel release version of the target host. You can fill `$TARGET_KERNEL` with the output of the command: `"$(uname -r)_$(uname -v | sed 's/#\([[:digit:]]\+\).*/\1/')"`, running it on a shell on the target host.
+
+for example:
+
+```shell
+make generate -e TARGET_DISTRO=arch -e TARGET_KERNEL=6.3.4-arch1-1
+```
+
+> Available values for `$TARGET_DISTRO` can be found [here](https://github.com/falcosecurity/driverkit/tree/master/pkg/driverbuilder/builder).
+
+### Advanced settings
+
+> Besides `$TARGET_DISTRO` and `$TARGET_KERNEL`, you can find all the supported filters [here](https://github.com/falcosecurity/test-infra/blob/master/driverkit/README.md#available-make-targets), that help you to target specific kernel releases.
+
+## Propose the configuration
+
+Now you're ready to propose the configuation, by sending a pull request to the [test-infra](https://github.com/falcosecurity/test-infra) upstream repository, with the changes.
+
+When accepted, the configurations will be then consumed by [Driverkit](https://github.com/falcosecurity/driverkit) in the [CI pipeline](https://github.com/falcosecurity/test-infra/tree/master/config/jobs/build-drivers), through the [Driverkit Build Grid](https://github.com/falcosecurity/test-infra/tree/master/driverkit#driverkit-build-grid).
+
+## Examples
+
+You can find examples of configurations in the repository at the paths structured as follows:
+
+  `driverkit/config/<driver version>/<architecture>/<linux distribution>_<kernel release name>_<kernel build version>.yaml`.
+
+For example this is a configuration for Ubuntu generic kernel, release `6.3.0-4-generic`, for the driver `5.0.1`:
+
+```yml
+kernelversion: 4
+kernelrelease: 6.3.0-4-generic
+target: ubuntu-generic
+architecture: amd64
+output:
+  module: output/5.0.1+driver/x86_64/falco_ubuntu-generic_6.3.0-4-generic_4.ko
+  probe: output/5.0.1+driver/x86_64/falco_ubuntu-generic_6.3.0-4-generic_4.o
+kernelurls: ["http://security.ubuntu.com/ubuntu/pool/main/l/linux/linux-headers-6.3.0-4-generic_6.3.0-4.4_amd64.deb","http://mirrors.edge.kernel.org/ubuntu/pool/main/l/linux/linux-headers-6.3.0-4-generic_6.3.0-4.4_amd64.deb","http://mirrors.edge.kernel.org/ubuntu/pool/main/l/linux/linux-headers-6.3.0-4_6.3.0-4.4_all.deb","http://security.ubuntu.com/ubuntu/pool/main/l/linux/linux-headers-6.3.0-4_6.3.0-4.4_all.deb"]
```

Comment:
What about also adding a note about `kernelurls` (that are not automatically generated by the `make generate` dbg target).
Ie: it would help if people would start adding sensible `kernelurls` param to newly added dbg config!

Reply:
I like the idea, I'll do it! thx @FedeDP